src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    addOne: () => {
+      obj.result++;
+    },
+
+    reset: () => {
+      obj.result = 0;
+
+      return obj;
```

Comment:
The `reset` method correctly resets the `result` to 0 and returns the object for chaining. Good job following the task requirements.